other-builds/ndkbuild/bitmap-plasma/app/build.gradle

```diff
@@ -0,0 +1,38 @@
+apply plugin: 'com.android.application'
+
+// pointing to cmake's source code for the same project
+def REMOTE_PROJ_ROOT = '../../../../' + rootProject.getName() + '/' +
+                        project.getName() + '/src/main'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "23.0.3"
+
+    defaultConfig {
+        applicationId "com.example.plasma"
+        minSdkVersion 8
+        targetSdkVersion 8
+        ndkConfig.abiFilters 'armeabi'
```

Comment:
Oh, ndk.abiFilters does not work ( it errored out ). ndkConfig.abiFilters works ( I think this is residual from android studio 1.3 implementation ).  I filed a bug for it will search and send to you